### ISSUE TITLE
Could you tell us what paper or algorithm your mask_refine.ccp is based on

### ISSUE THREAD

**Issue by wengjiyao** [xiongzhu666/Sky-Segmentation-and-Post-processing#8]
Could you tell us what the paper mentioned in your mask_refine.ccp and what algorithms it implements?

**Comment by xiongzhu666** [comment]
> Could you tell us what the paper mentioned in your mask_refine.ccp and what algorithms it implements?
> 
> Thank you!

 https://arxiv.org/abs/2006.10172.  I had writen in the repo

**Comment by calebhemara** [comment]
Will you release onnx of mask_refine with pre-trained weights in the future? Thanks

**Comment by xiongzhu666** [comment]
> Will you release onnx of mask_refine with pre-trained weights in the future? Thanks

Mask refine is not based on deep-learning，it is just based on weight guided filter as the paper describe

**Comment by calebhemara** [comment]
I see, thank you !
